packages/grafana-data/src/themes/types.ts

```diff
@@ -25,6 +26,7 @@ export interface GrafanaTheme2 {
   typography: ThemeTypography;
   zIndex: ThemeZIndices;
   shadows: ThemeShadows;
+  vizColors: ThemeVizColors;
```

Comment:
could this be a subtype to colors instead and called `visualizations` so `colors.visualizations.xyz`

Reply:
Good idea, but not sure. The thinking is that visualization colors are sort of separate but dependent on theme color mode. So líke that they are defined separately and only take in the ThemeColors as a foundation. It's going to be a bit trickier to have this be part of ThemeColors and also depend on it when it's defined

Reply:
Actually, I prefer the vizColors separation rather than nesting those under colors. I feel you @torkelo with the concerns separation here and conceptually I think the visualization colors are a different beast.

Reply:
Thinking about this more I think I would consider nesting, but, the other way around.

`theme.vizColors` would become `theme.visualization.colors`. My thinking here is that this approach would allow us to namespace and extend viz specific things like for instance typography that we may optimize in the future rather than using the standard Arial or whatever. Not 100% convinced, but I think namespacing this straight away will make it easier to make any changes.